### PR TITLE
Some fixes for gmod_wire_hud_indicator_2 and such

### PR DESCRIPTION
**Pull request by UAVXP**

### CODE REVIEW

**Comment by oldmud0** [comment]
Better commit messages would be nice.

File: lua/entities/base_gui_panel/cl_init.lua [hunk 1]
```diff
@@ -83,12 +83,15 @@ function ENT:Draw()
 end
 
 function endDrawTimer(ent)
+	if !IsValid( ent ) then return end
```
Comment (oldmud0) [inline lua/entities/base_gui_panel/cl_init.lua]: For new code onwards, `not` and other official Lua syntax must be used. `!` is nonstandard GLua syntax and is in the process of being eradicated completely from the project.

**Comment by Divran** [comment]
Alright, UAVXP just asked me to merge this. I haven't tested it, but I can't see anything wrong by looking at the code, so I say it's fine to merge. I'll do it later today if no one objects

**Comment by Divran** [comment]
Ok I'm gonna trust that UAVXP tested it. Since this is wire extras and not the main repo, it's not as important really. Also considering this thing didn't work at all before this PR, it can't not be an improvement over what we had before

**Pull request merged**